### PR TITLE
[TWINFACES-61] List of links in selected twin

### PR DESCRIPTION
**Pull request by oleg-lego-lego**

### CODE REVIEW

File: src/lib/api/api-groups/twin-api.ts [hunk 1]
```diff
@@ -34,7 +34,45 @@ export function createTwinApi(settings: ApiSettings) {
...
+                header: getApiDomainHeaders(settings),
+                query: {
+                    showTwin2TwinLinkMode: "DETAILED",
+                    offset: 0,
+                    limit: 10,
+                }
+            },
+            body: [
+                {
+                    twinIdList: ["1fc37cd5-3a6e-4529-8ef3-e196242b1c38"]
```
Comment (pihaizer) [inline src/lib/api/api-groups/twin-api.ts:69]: Please use links from the parameter, not hard coded

File: src/components/auto-dialog.tsx [hunk 2]
```diff
@@ -97,7 +97,7 @@ export function AutoDialog({open, onOpenChange, settings}: EditFieldDialogProps)
                         {error}
                     </Alert>}
 
-                    <DialogFooter className="sm:justify-end">
+                    <DialogFooter className="sm:justify-end pb-4">
```
Comment (pihaizer) [inline src/components/auto-dialog.tsx:100]: Why this change?

**Review by pihaizer — CHANGES_REQUESTED** [review]

File: src/components/auto-dialog.tsx (hunk 2 above)
Comment (oleg-lego-lego) [reply src/components/auto-dialog.tsx:100]: added indentation for the button, it was pressed to the bottom of the window

**Review by pihaizer — APPROVED** [review]
